src/app/tests/suites/ciTests.json

```diff
@@ -12,14 +12,24 @@
         "Test_TC_ACL_2_9",
         "Test_TC_ACL_2_10"
     ],
-    "AccessControlEnforcement": ["Test_TC_ACE_1_1", "Test_TC_ACE_1_5"],
-    "BooleanState": ["Test_TC_BOOL_1_1", "Test_TC_BOOL_2_1"],
-    "Binding": ["Test_TC_BIND_1_1"],
+    "AccessControlEnforcement": [
```

Comment:
@duyen-tran let's remove the changes from this file for now. Cecille clarified the CI build is caused by something else.